capstone-frontend/src/components/ListPage.jsx

```diff
@@ -77,33 +80,53 @@ class ListPage extends Component {
       :
       (this.state.empty
         ?
-        (<h1 className="text-center mt-4">Booklist has No Books</h1>)
+        (
+          <div>
+            <h2 className="mt-3">{this.state.bookList.name}</h2>
+            <hr style={{ border: "1px solid #ccc" }} />
```

Comment:
You should move this styling to a CSS class and use a shared class name for your `hr` elements

Reply:
Done.